models/main.py

```diff
@@ -0,0 +1,74 @@
+from est.calc import constr
```

Comment:
This needs to be turned into a function so you can iterate through. Also separate out the plotting functionality into separate sub-module.